roles/downstream-setup/tasks/yum_repos.yml

```diff
@@ -0,0 +1,11 @@
+---
+- fail:
```

Comment:
I couldn't decide if failing here or adding a when: yum_repos|length > 0 to the include was a better approach.

If we want this playbook to have multiple snippets somebody building qa-suite yaml could leverage then we'd need some way for that user to chose which snippet to run.  I see two ways. 1) add a when clause to each include to check for the vars needed for the included tasks.  If the vars aren't defined in the yaml, it won't be ran. or 2) support --tags and the user will need to add tags in their yaml specifying which tasks to run.

@zmc @alfredodeza @ktdreyer thoughts?